cooperator/__manifest__.py

```diff
@@ -17,7 +17,7 @@
         "product",
         "partner_firstname",
         "partner_contact_birthdate",
-        "partner_contact_address",
+        "partner_contact_address_default",
```

Comment:
Are you sure about this change ?

Reply:
Ah non j'ai confondu le module de l'OCA avec celui dans addons. Je le retire.

Reply:
je retest

Reply:
OK c'est bon !